### PR TITLE
Registry setup

### PR DESCRIPTION
**Pull request by David176**

### CODE REVIEW

File: services/registry/Migrations/DbInitializer.cs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            context.Database.EnsureCreated();
+
+            if (context.Shares.Any())
+            {
+                return; // Database already seeded
+            }
+
+            var shares = new[]
+            {
+                new ShareModel{TickerSymbol = "NVO", Owner = 1},
```
Comment (bjornnorgaard) [inline services/registry/Migrations/DbInitializer.cs]: Bruge for-løkke.

File: services/registry/Models/BuyModel.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Registry.Models.ViewModel
+{
+    public class BuyModel
+    {
+        public int SellerId { get; set; }
+        public int Quantity { get; set; }
+        public int BuyerId { get; set; }
+        public string TickerSymbol { get; set; }
+    }
+}
```
Comment (bjornnorgaard) [inline services/registry/Models/BuyModel.cs]: Mangler newline til sidst i filen

**Review by bjornnorgaard — CHANGES_REQUESTED** [review]

File: services/registry/Migrations/DbInitializer.cs [hunk 3]
```diff
@@ -13,132 +14,19 @@ public static void Initialize(ShareContext context)
...
+                sharesList.Add(new ShareModel { TickerSymbol = "NVO", Owner = 1 });
+                sharesList.Add(new ShareModel { TickerSymbol = "GOOG", Owner = 2 });
+                sharesList.Add(new ShareModel { TickerSymbol = "VWS", Owner = 3 });
+                sharesList.Add(new ShareModel { TickerSymbol = "DANSKE", Owner = 4 });
+                sharesList.Add(new ShareModel { TickerSymbol = "MAERSK A", Owner = 5 });
+                sharesList.Add(new ShareModel { TickerSymbol = "DENERG", Owner = 6 });
+            }
 
-            foreach (var share in shares)
+            foreach (var share in sharesList)
```
Comment (bjornnorgaard) [inline services/registry/Migrations/DbInitializer.cs]: brug flertal

**Review by bjornnorgaard — DISMISSED** [review]

File: services/registry/Models/BuyModel.cs (hunk 2 above)
Comment (bjornnorgaard) [reply services/registry/Models/BuyModel.cs]: Denne mangler stadig

Comment (David176) [reply services/registry/Models/BuyModel.cs]: Nej, hent det ned, der er newline

File: services/registry/Migrations/DbInitializer.cs (hunk 3 above)
Comment (David176) [reply services/registry/Migrations/DbInitializer.cs]: No, this is correct

File: services/registry/Models/BuyModel.cs (hunk 2 above)
Comment (bjornnorgaard) [reply services/registry/Models/BuyModel.cs]: Improved review by 10%.

File: services/registry/Migrations/DbInitializer.cs (hunk 3 above)
Comment (bjornnorgaard) [reply services/registry/Migrations/DbInitializer.cs]: Another review improvement. Additional 11% has been added to quality of previously created tasks.

**Comment by bjornnorgaard** [comment]
Now fix.

File: services/registry/Models/BuyModel.cs (hunk 2 above)
Comment (bjornnorgaard) [reply services/registry/Models/BuyModel.cs]: Git derper, du har ret.

File: services/registry/Migrations/DbInitializer.cs (hunk 3 above)
Comment (bjornnorgaard) [reply services/registry/Migrations/DbInitializer.cs]: Vi er ligeglade med at det er en `List<>.` Det eneste relevante er: 
1. At den indeholder `Shares` og
2. At der er en collection.

**Review by dentych — APPROVED** [review]
such amaze. master code

**Pull request merged**